### PR TITLE
Add TEMPLATES and WORKFLOWS

### PR DESCRIPTION
**Pull request by ReenigneArcher**
## Description

Add .github folder:
- ISSUE_TEMPLATE
- workflows
- etc.

### Screenshot

Include screenshots if the changes are UI-related.

### Issues Fixed or Closed

- Fixes #(issue)

## Type of Change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated the documentation blocks for new or existing components

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+**Screenshots**
+<!-- If applicable, add screenshots to help explain your problem. -->
+
+**Host**
+<!-- Please complete the following information -->
+ - OS: [e.g. Windows, Ubuntu]
+ - Architecture [e.g. 32 bit, 64 bit, arm]
+ - Version [e.g. 0.11.1]
+
```
Comment (TheElixZammuto) [inline .github/ISSUE_TEMPLATE/bug-report.md:31]: I would add here other important informations for GPU and host. On Windows the driver version and the GPU Vendor (Intel, AMD, Nvidia and the model). On Linux the capture method (PipeWire/KVM/X11) and the appropriate info (like the Mesa Version)

Comment (ReenigneArcher) [reply .github/ISSUE_TEMPLATE/bug-report.md:31]: How about this?

```yaml
**Host**
<!-- Please complete the following information -->
 - OS: [e.g. Windows, Linux, Mac... include build/distro details]
 - Architecture: [e.g. 32 bit, 64 bit, arm]
 - Version: [e.g. 0.11.1]
 - GPU Type: [e.g. Intel, AMD, Nvidia]
 - GPU Model:
 - GPU Driver/Mesa Version:
 - Capture method (Linux only): [e.g. PipeWire/KVM/X11]
```

Comment (TheElixZammuto) [reply .github/ISSUE_TEMPLATE/bug-report.md:31]: Yup, I like this!

Comment (ReenigneArcher) [reply .github/ISSUE_TEMPLATE/bug-report.md:35]: @TheElixZammuto we good to merge this?

**Pull request merged**